### PR TITLE
Move Manually Created SES Receipt Rules into Terraform

### PR DESCRIPTION
**Pull request by emileswarts**
We want this to be managed by Terraform as it involves less manual
effort moving forward.

Creating the rule set itself is still manual as this causes errors with
Terraform applying this accross multiple environments.

Delete the unused test rules which belonged to an inactive test rule set.
Recreate the rules that were manually created.

This doesn't make the new rule set active, it only creates a copy which
can be made active once we are happy.

### CODE REVIEW

File: govwifi-emails/emails.tf [hunk 1]
```diff
@@ -1,28 +1,53 @@
...
 
   depends_on = [
-    "aws_ses_receipt_rule_set.main-ses-ruleset",
     "aws_sns_topic.govwifi-email-notifications",
     "aws_s3_bucket.emailbucket",
+    "aws_ses_receipt_rule.user-signup-rule",
   ]
 
   recipients = [
     "enrol@${var.Env-Subdomain}.service.gov.uk",
```
Comment (adrianclay) [inline govwifi-emails/emails.tf]: We can remove enrol, enroll and signup from this rule now.  Given they are being processed by the rule above.

Comment (emileswarts) [reply govwifi-emails/emails.tf]: Yeah noticed that but wanted to preserve current functionality and address that in a future PR.
Just to minimise risk on activating this new rule set for now.

**Comment by emileswarts** [comment]
Have opened an issue regarding this SES Rule Set being a manual step.

File: govwifi-emails/emails.tf (hunk 1 above)
Comment (adrianclay) [reply govwifi-emails/emails.tf]: Fair enough.

**Pull request merged**